Bandwidth.Iris/Model/PortIn.cs

```diff
@@ -143,6 +143,23 @@ public async Task<Note[]> GetNotes()
                     .List;
         }
 
+        public Task<LnpResponseWrapper> GetPortIns(string accountId, DateTime enddate, DateTime startdate, string pon, string status, string tn, int page = 1, int size = 300)
+        {
+
+            Dictionary<string, object> query = new Dictionary<string, object>();
+
+            query.Add("page", page);
+            query.Add("size", size);
+            
+            if (enddate != null) query.Add("enddate", enddate);
+            if (startdate != null) query.Add("startdate", startdate);
+            if (pon != null) query.Add("pon", pon);
+            if (status != null) query.Add("status", status);
+            if (tn != null) query.Add("tn", tn);
```

Comment:
The docs also have a query param for `date`. Is that needed here?

Reply:
Yes, adding